packages/bloc/lib/src/bloc.dart

```diff
@@ -25,7 +25,7 @@ abstract class Bloc<Event, State> extends Cubit<State>
   /// The current [BlocObserver].
   static BlocObserver observer = BlocObserver();
 
-  final _eventController = StreamController<Event>.broadcast();
```

Comment:
The event `StreamController` is only ever subscribed to once, in `_bindEventsToStates ` which is called by the constructor, so there seems to be no need for using a broadcast stream.

Reply:
Great catch! Thanks for the improvement 👍